README.md

```diff
@@ -864,22 +851,28 @@ For new bug reports, feature requests and support:  Please open an issue in Gith
 [TOC](#TOC)
 <div id="4">
 
-## 4 Running the test suite
+## 4 Test Suite
 
-#### Different kinds of tests:
+The test suite is implemented using CTest.  CMake adds the target 'test' to the build.  You can run ctest with the command `$ cmake --build . --target test`, or simply with `$ ctest`.  The build creates 6 tests: bashTests, bugfixes, lensTest, tiffTests, unit_tests and versionTest.  You can run all tests or a subset.
```

Comment:
Can we also rename just the test name (not the executable) from `unit_tests` to `unitTests` everywhere to have the same style? This involves changing just the first mention [here](https://github.com/Exiv2/exiv2/blob/29cc98118003a9f1c632bde9bd706f9483de4e8b/unitTests/CMakeLists.txt#L54) I think?

Reply:
I will be delighted to change that.

I like things to be consistent and I've inherited inconsistency.

```
905 rmills@rmillsmm-local:~/gnu/github/exiv2/main $ finder CMakeLists.txt | xargs grep add_test
./CMakeLists.txt:        add_test(NAME bashTests
./CMakeLists.txt:        add_test(NAME bugfixes
./CMakeLists.txt:        add_test(NAME lensTests
./CMakeLists.txt:        add_test(NAME tiffTests
./CMakeLists.txt:        add_test(NAME versionTest
./unitTests/CMakeLists.txt:add_test(NAME unit_tests COMMAND unit_tests)
906 rmills@rmillsmm-local:~/gnu/github/exiv2/main $ 
```

I will not change the name of the directory `tests/bugfixes` or `tests/bash_tests`, however I change the test name to bugfixTests.  This will be much nicer:

| Name               | Language  | Location    | Command<br>_(in build directory)_ | CMake Option to Build          |
|:--                 |:--        |:--                      |:--                    |:--                             |
| bashTests          | python    | tests/bash\_tests       | $ ctest -R bash       | -DEXIV2\_BUILD\_SAMPLES=On     |
| bugfixTests          | python    | tests/bugfixes          | $ ctest -R bugfix   | -DEXIV2\_BUILD\_SAMPLES=On     |
| LensTests           | C++       | tests/lensTests        | $ ctest -R lens       | -DEXIV2\_BUILD\_SAMPLES=On     |
| tiffTests          | python    | tests/tiffTests         | $ ctest -R tiff       | -DEXIV2\_BUILD\_SAMPLES=On     |
| unitTests        | C++       | unitTests               | $ ctest -R unit       | -DEXIV2\_BUILD\_UNIT\_TESTS=On |
| versionTests        | C++       | src/version.cpp         | $ ctest -R version    | Always in library              |

Reply:
Nice(r)! Just need that leading small caps on "lensTests" ;)